src/models/request.js

```diff
@@ -39,13 +39,19 @@ module.exports = (sequelize, DataTypes) => {
       allowNull: false,
       defaultValue: false,
     },
+    multiCity: {
+      type: DataTypes.BOOLEAN,
+      allowNull: false,
+      defaultValue: false,
```

Comment:
I think this should also be in the sequelize file

Reply:
will add to the migration file as suggested, thanks.